requirements-test.txt

```diff
@@ -1,3 +1,3 @@
 coverage==5.*
-flake8==3.7.9
+flake8==6.*
```

Comment:
Can we use `5.*` to avoid workaround in CI? The latest version of 5 series `5.0.4` catches the flake error.